packages/mollie-integration/src/mollie-payment-strategy.ts

```diff
@@ -300,18 +283,18 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         );
     }
 
-    private _mountCardVerificationfields(formOptions: HostedFormOptions): Promise<HostedForm> {
+    private mountCardVerificationfields(formOptions: HostedFormOptions): Promise<HostedForm> {
         /* eslint-disable */
```

Comment:
why we disable esLint there?

Reply:
When eslint is enabled we get `Promise executor functions should not be async` and `Promise returned in function argument where a void return was expected` exceptions.